src/components/jobComponents/createJob.tsx

```diff
@@ -119,93 +124,97 @@ const CreateJob: React.FC = () => {
       style={{ maxWidth: '400px', textAlign: 'center' }}
       id="createJobForm"
     >
-      <h4>Create New Job</h4>
-      <div className="input-field fsField">
-        <input
-          value={name}
-          onChange={e => handleChange(e, setName, 'nameLabel')}
+      <h4>Create a New Job</h4>
+      <form className={classes.root} noValidate autoComplete="off">
+        <TextField
           id="name"
-          autoComplete="off"
-          className={name.length ? (name.length > 3 ? 'valid' : 'invalid') : ''}
+          label="Name"
+          value={name}
+          error={!!name.length && name.length < 4}
+          onChange={e => setName(e.target.value)}
```

Comment:
i think if you have name.length < 4 you are also cover !!name.length

Reply:
with !!name.length && ... it means if there's nothing in the field it won't show the error. then in handleSubmit it checks if name is truthy, so an empty string won't pass.